language/evm/move-to-yul/src/native_functions.rs

```diff
@@ -104,4 +106,28 @@ impl NativeFunctions {
             );
         })
     }
+
+    /// Define vector functions for a specific instantiation.
+    fn define_vector_functions(&mut self, ctx: &Context) {
```

Comment:
I would move this to the new vector module.

Reply:
Will do so in later PRs.